pandas/core/generic.py

```diff
@@ -2799,13 +2800,22 @@ def to_sql(
         schema : str, optional
             Specify the schema (if database flavor supports this). If None, use
             default schema.
-        if_exists : {'fail', 'replace', 'append'}, default 'fail'
+        if_exists : {'fail', 'replace', 'append'},\
+        default 'fail'
```

Comment:
why are you u using row-continuation here? just leave it

can consider deprecating this and change the name to `if_table_exists` to be more obvious (but orthogonal)